python/cuxfilter/dashboard.py

```diff
@@ -416,6 +441,9 @@ def app(self, notebook_url=DEFAULT_NOTEBOOK_URL, port: int = 0):
             Default is random open port. Recommended to set this value if
             running jupyter remotely and only few ports are exposed.
 
+        environment: str, optional, default jupyter-notebook,
+            options: jupyter-notebook, jupyterlab, jupyterhub
```

Comment:
Is there a better parameter name than "environment"? That word already has lots of other uses. Some ideas
```
jupyter = classic | lab | hub
notebook = classic | jupyterlab | jupyterhub
```
Or 
```
service_proxy = none | jupyterlab
```
?

Reply:
ah, I agree with you on this, so technically, we only have two scenarios, adding the prefix for jupyterhub, or not adding it for jupyterlab/classic, given that, I think should work:

```python
service_proxy = none(for classic/lab) | jupyterhub
```
does this seem fine?

Reply:
replaced with service_proxy, can you have a look and see if it looks good?